src/scratch.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
This is fine but I think we should just add a little comment to say this is for testing the database connection and maybe rename the file to "database_connection_test.py" or something like that. The other thing is that I would prefer if this lived in the integration_tests folder. The idea with the `src` folder is that is it for the python modules that make up the SDS and their corresponding unit tests.

Reply:
Perhaps I should have said something, at least on Slack. It will be gone with the next ticket. Can you live with it  for this one, please? I will look at the tests and getting thing everything through the pipeline in the next ticket.

Reply:
That's fine, if the plan is to remove it in the future.